tests/unit/views/services/camera_test.js

```diff
@@ -0,0 +1,112 @@
+import Rx from 'rxjs';
+import React from 'components/react';
+
+import CameraServiceView from 'js/views/services/camera';
+
+// See capabilities at https://facebook.github.io/react/docs/test-utils.html
+const TestUtils = React.addons.TestUtils;
+
+function waitFor(fn) {
+  return new Promise((resolve) => {
+    Rx.Observable.interval(1)
+      .filter(() => fn())
+      .take(1)
+      .subscribe(resolve);
+  });
+}
+
+describe('Camera service view tests', function () {
+  let foxboxStub, serviceStub, component;
+
+  before(function() {
+    foxboxStub = sinon.stub({
+      performSetOperation: () => {},
+      performGetOperation: () => {}
+    });
+
+    serviceStub = sinon.stub({
+      setters: {
+        'fake-setter': {
+          kind: 'Fake'
+        },
+
+        'setter-id': {
+          kind: 'TakeSnapshot'
+        }
+      },
+      getters: {
+        'fake-getter': {
+          kind: { kind: 'fake' }
+        },
+
+        'getter-id': {
+          kind: { kind: 'latest image' }
+        }
+      },
+      properties: {
+        name: 'service-name'
+      }
+    });
+
+    component = TestUtils.renderIntoDocument(
+      <CameraServiceView service={serviceStub} foxbox={foxboxStub} />
+    );
+  });
+
+  it('Renders itself in the correct state', function () {
```

Comment:
That makes sense, let's not write too much code that'll be throw away soon.
We'll have to implement the new design at some point and I expect it'll change a lot of things.